app/web/src/components/Navigation/Navigation.tsx

```diff
@@ -163,6 +256,57 @@ export default function Navigation() {
     </div>
   );
 
+  const menuItems = (
+    <div>
```

Comment:
I thought that too but i think it's for a good reason, here's the error thrown by material-ui

![01](https://user-images.githubusercontent.com/62873665/145074320-0ae3805c-8a2b-49d5-a3e1-7caec5d1c240.png)

Reply:
How about an array then?

Reply:
Using this as an array is quite important here - I just had a play around and it's because of the outer div here, it's preventing us from going between the items in the menu when you are using the keyboard arrows, since the `div` somehow became the only focusable thing...